src/commands/test/test_environment_exceptions.py

```diff
@@ -10,6 +11,28 @@ class MissingExceptException(ReportedException):
     pass
 
 
+class ExceptMissmatchException(ReportedException):
+    def __init__(
+        self,
+        expected_name: Union[str, None],
+        expected_message: Union[str, None],
```

Comment:
> You can use the [Optional](https://docs.python.org/3/library/typing.html#typing.Optional) type modifier to define a type variant that allows None, such as Optional[int] (Optional[X] is the preferred shorthand for Union[X, None])

```suggestion
        expected_name: Optional[str],
        expected_message: Optional[str],
```

Reply:
I thought optional can only be a keyword arg for some reason